app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,24 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
```

Comment:
@amaurygonzalez i am not sure why are we doing group_by on description here, did you mean to use rate_type. Description can be any text entered by user, where as rate_type can be "Compute" or "Storage"